src/loggers.jl

```diff
@@ -71,6 +73,99 @@ function log_property!(logger::CoordinateLogger, s::Simulation, step_n::Integer)
     end
 end
 
+"""
+    ForceLogger(n_steps; dims=3)
+
+Log the forces throughout a simulation.
+"""
+struct ForceLogger{T} <: Logger
+    n_steps::Int
+    forces::Vector{Vector{T}}
+end
+
+function ForceLogger(T, n_steps::Integer; dims::Integer=3)
+    return ForceLogger(n_steps,
+                            Array{SArray{Tuple{dims}, T, 1, dims}, 1}[])
```

Comment:
You could just use `Vector{SVector{dims}}` here for readability.
```suggestion
                            Vector{SVector{dims}}[])
```